src/test/regress/columnar_am_schedule

```diff
@@ -0,0 +1,18 @@
+test: am_create
+test: am_load
+test: am_query
+test: am_analyze
+test: am_data_types
+test: am_functions
+test: am_drop
+test: am_insert
+test: am_copyto
+test: am_alter
+test: am_rollback
+test: am_truncate
+test: am_vacuum
+test: am_clean
+test: am_block_filtering
+test: am_join
+test: am_trigger
+test: am_tableoptions
```

Comment:
random note: we should probably add some non-superuser tests

Reply:
Once we have the repo's merged we can invest in adding tests with the test infra we have within Citus